api/.bash_aliases

```diff
@@ -0,0 +1 @@
+export DB_URI="mongodb://127.0.0.1:27017/snake"
```

Comment:
You should be able to delete this file we do not need it, this was when we were in the wrong dir